### ISSUE TITLE
Symlinked content has wrong __url

### ISSUE THREAD

**Issue by kennu** [MoOx/phenomic#800]
I think this is a bug. In my Phenomic content folder I have a symlink to a subfolder somewhere else (actually under a Gatsby project that I'm migrating):

```
arp -> ../../frontend/pages/arp
```

When I access arp/index.md in Phenomic, it has an invalid __url field:

```
__url: "/frontend/pages/arp/"
```

I would expect it to be:

```
__url: "/arp/"
```

**Comment by MoOx** [comment]
Hum, interesting.
So your content is in a parent folder and you linked it in content/arp?

**Comment by thangngoc89** [comment]
@kennu we rely on Webpack for path resolution. Could you help me debug this? 

I need to know about the webpackInstance.resourcePath [here](https://github.com/MoOx/phenomic/blob/a15fdef6ee8e9895881fabccd096978bd5609907/src/loader/index.js#L29) 

Could you log it and post it here?

**Comment by kennu** [comment]
To clarify the structure, I have two projects side-by side: frontend (Gatsby) and frontend2 (Phenomic):

    frontend/pages/arp/index.md
    frontend2/content/arp -> ../../frontend/pages/arp

So my goal is to share the .md content files while migrating.

**Comment by MoOx** [comment]
Thank @kennu, this should be enough for us to track down this issue.

**Comment by MoOx** [comment]
@kennu I took a look and I would like to know what "context" option are you passing to phenomic folder?
The thing is: webpack is resolving the path, and we are making it relative to the context, so we are creating the "../../frontend/pages" part, and them we join this with the basepath of your website url (so probably / in your case). and joining / with "../../whatever" is dropping the ../.. part.
Not sure how to fix this...

**Comment by kennu** [comment]
@MoOx I think it's the default generated for a new project, this is in webpack.config.babel.js:

    phenomic: {
      context: path.join(__dirname, config.source),

I'm not sure why it's even necessary to treat symlinks specially. Normally they would be resolved automatically at the OS level..

**Comment by MoOx** [comment]
The thing is: we transform the full absolute (resolved) path as a relative path to the content. That why you end up with `../../frontend/pages/arp`. And since we join this with the basepath of the website, to have absolute uri the .. are ditched (`/../../frontend/pages/arg` can't be...).

So in the first place maybe we should not resolve the path completely? But webpack does this... We have no control.

One workaround will be to use a specific loader section just for this files, like this

```js
{
  test: /\.md$/,
  loader: phenomicLoader
  exclude: /frontend\/pages\/arp/,
  query: {
    context: path.join(__dirname, config.source),
  }
},
{
  test: /\.md$/,
  loader: phenomicLoader
  include: /frontend\/pages\/arp/,
  query: {
    context: path.join(__dirname, "..", "..", "frontend", "pages"),
  }
}
```

Not really straight foward, but not sure how we can make this better since we need to make filepath relative to context to "guess" urls.

**Comment by bloodyowl** [comment]
Going to be fixed with #925 (moving to user-land)

**Comment by MoOx** [comment]
Might be fixed with v1.
